docs/pages/versions/unversioned/sdk/sqlite.mdx

```diff
@@ -101,8 +109,4 @@ console.log('Foreign keys turned on');
 
 ## API
 
```

Comment:
````suggestion

```js
import * as SQLite from 'expo-sqlite';
```
````

We use this pattern for every library in our API references.